games-util/springlobby/metadata.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>totocoq@yahoo.fr</email>
+		<name>Thomas Coquelin</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<upstream>
+		<remote-id type="github">springlobby/springlobby</remote-id>
+		<bugs-to>https://github.com/springlobby/springlobby/issues</bugs-to>
+	</upstream>
+	<use>
+		<flag name="translation">Enable Translations support</flag>
+	</use>
+	<longdescription lang="en">Springlobby client</longdescription>
```

Comment:
This is not a *long* description. Use this tag only if you have something very long that can't fit in `DESCRIPTION` in ebuild.

Reply:
Removed